gc/base/MemorySubSpaceUniSpace.cpp

```diff
@@ -573,35 +579,35 @@ uintptr_t
 MM_MemorySubSpaceUniSpace::checkForRatioExpand(MM_EnvironmentBase *env, uintptr_t bytesRequired)
 {
 	Trc_MM_MemorySubSpaceUniSpace_checkForRatioExpand_Entry(env->getLanguageVMThread(), bytesRequired);
-	
+
 	uint32_t gcPercentage;
-	uintptr_t currentFree, maxFree;
+	uintptr_t currentFree, maxFree, heapFreeMaximumHeuristicMultiplier;
```

Comment:
Again try to keep variable initialized. Either bring down the definition to init site, or if it does not make sense, init them to some neutral variable (0)